source/logger.js

```diff
@@ -33,19 +49,61 @@ Space.Logger = Space.Object.extend(Space, 'Logger', {
     }
   },
 
-  info(message) {
-    if (this.shouldLog()) this._logger.info(message);
+  debug(message, meta) {
```

Comment:
There is a lot of repetition (boilerplate) going on for wrapping the `this._logger` api, so it might be better to move that into a `onExtending` hook where you can add them to the prototype in one loop.

You can see how to use that here: https://github.com/meteor-space/base/blob/develop/tests/unit/object.unit.coffee#L41
The `onExtending` method is called once after the class has been constructed and the context (`this` value) of the method is the class it self :wink: